test/controllers/songs_controller_test.rb

```diff
@@ -1,19 +1,19 @@
 require 'test_helper'
 
 class SongsControllerTest < ActionController::TestCase
-  # test "should get index" do
-  #   get :index
-  #   assert_response :success
-  # end
+  test "should get index" do
+    get :index
+    assert_response :success
+  end
 
-  # test "should get new" do
-  #   get :new
-  #   assert_response :success
-  # end
-
-  # test "should get edit" do
-  #   get :edit
-  #   assert_response :success
-  # end
+  test "index should retrieve relevant search results" do
+    get :index, sSearch: 'hand', format: :json, xhr: true
+    songs = JSON.parse(@response.body)['data']
+    assert_equal(songs.length, 2)
```

Comment:
Also please write one more test for index with empty params (simulating initial page load).